meson.build

```diff
@@ -1,28 +1,31 @@
 project('pmt', 'cpp',
   version : '0.0.2',
-  meson_version: '>=0.52.0',
+  meson_version: '>=0.63.0',
   license : 'GPLv3',
   default_options : ['cpp_std=c++20'])
 
 cc = meson.get_compiler('cpp')
 rt_dep = cc.find_library('rt', required : false)
 c_available = add_languages('c', required : true)
-python3_dep = dependency('python3', required : get_option('enable_python'))
+
+if (get_option('enable_python'))
```

Comment:
Any reason you are using unnecessary parentheses here? It's not part of the `if` grammar, and you don't use it in lines such as `if pybind11_dep.version().version_compare('<2.4.4')`